### ISSUE TITLE
Rename glacier parameter `g_tt` to `g_ttm`

### ISSUE THREAD

**Issue by JoostBuitink** [Deltares/hydromt_wflow#309]
### HydroMT-Wflow version checks

- [x] I have checked that this issue has not already been reported.
- [x] I have checked that this bug exists on the latest version of HydroMT-Wflow.

### Reproducible Example

Given the changes in https://github.com/Deltares/Wflow.jl/pull/512, we should update the code and template .toml to reflects these changes

### Current behaviour

Parameter is called `g_tt`

### Desired behaviour

 Parameter should be called `g_ttm`

**Comment by vers-w** [comment]
Thanks for reporting this issue @JoostBuitink ! 

I think the required changes on the hydromt_wflow side affect all model input parameters/variables and parameters/output variables that we want to exchange when we implement standard names for these parameters/variables (https://github.com/Deltares/Wflow.jl/issues/481).

**Comment by JoostBuitink** [comment]
Yeah I realized that as well, but wanted to ensure that this one is registered here as well, as it was done as a separate PR. Was indeed thinking about how we should deal with these PRs, merging to master or to a wflow-v1 branch or something. Perhaps something to discuss at one of our standups!
